traffic_ops/traffic_ops_golang/deliveryservice/servers/servers_test.go

```diff
@@ -192,10 +194,13 @@ func getMockDSServers() []tc.DSServerV4 {
 		CDNName:      util.StrPtr("cdnTest"),
 		DomainName:   util.StrPtr("domain"),
 	}
-	srv := tc.DSServerV4{
+	srvV40 := tc.DSServerV40{
 		DSServerBaseV4:   base,
 		ServerInterfaces: &[]tc.ServerInterfaceInfoV40{}, // left empty because it must be written as json above since sqlmock does not support nested arrays
 	}
+	srv := tc.DSServerV4{
+		DSServerV40: srvV40,
+	}
```

Comment:
This part gets messy each time the struct version changes and has come up before (see https://github.com/apache/trafficcontrol/pull/4692#discussion_r426898413). If each field is assigned separately like
```go
	srv := tc.DSServerV4{}
	srv.ID = util.IntPtr(1)
	srv.Cachegroup = util.StrPtr("cgTest")
    // (and so on for the other fields)
	srv.ServerInterfaces = &[]tc.ServerInterfaceInfoV40{}
```
then that can be avoided.

Reply:
This was changed in `TestGetEligibleServers()` but not in `TestReadServers()`